themes/winamp/assets/fonts.css

```diff
@@ -0,0 +1,9 @@
+@font-face {
```

Comment:
Did you say this wasn't needed anymore? Just saving it for later?

Reply:
yes, we might still provide a way for them to link it